manala.locales/tasks/codes.yml

```diff
@@ -1,7 +1,14 @@
 ---
 
-- name: codes > Handle
+- name: Debian | codes > Handle
   locale_gen:
-    name:  "{{ (item) if (item is string) else (item.name) }}"
+    name:  "{{ (item) if (item is string) else (item.input+'.'+item.charmap) }}"
     state: "{{ item.state|default('present') }}"
   with_items: "{{ manala_locales_codes }}"
+  when: ansible_os_family == "Debian"
+
+- name: RedHat | codes > Handle
+  command: "localedef -v -c -i {{ item.input }} -f {{ item.charmap }} {{ item.input }}.{{ item.charmap }}"
```

Comment:
Can't the locale_gen ansible module handle locales instead of custom command ?

Reply:
Not working on redhat when I tested it